packages/gatsby-plugin-mdx/gatsby/source-nodes.js

```diff
@@ -186,19 +186,22 @@ module.exports = (
           }
           const { mdast } = await processMDX({ node: mdxNode })
 
-          const excerptNodes = []
+          let excerptNodes = []
```

Comment:
Why did you change this? I guess you literally copy/pasted the changes from the other PR? Please undo this line.

Reply:
(I've reverted this change from the other PR as well)